src/etcd.js

```diff
@@ -13,15 +14,22 @@ function applyChange (config, change) {
   let changed = false
   const [key, name, group] = getKey(config.prefix, change.node.key)
   const value = change.node.value
-  const level = group ? 4 : (name ? 3 : 2)
+  let level = 2
```

Comment:
Split this out to make the logic more clear and also to make it easier to see where coverage was required.